### ISSUE TITLE
IE11 Support?

### ISSUE THREAD

**Issue by stramel** [StartPolymer/s-circle-progress#6]
What are the supported browsers? AFAIK, IE11 doesn't support svg animations.

**Comment by JosefJezek** [comment]
Where is problem?

![image](https://user-images.githubusercontent.com/2110319/31509434-c403a55a-af81-11e7-8cf7-99d595576775.png)

**Comment by stramel** [comment]
Yeah, it does render and changing the value does change the amount filled. However, the animation doesn't work in IE11 it just jumps to the new value.

I just knew that a lot of people have an issue with SVG animations in IE11 since it isn't supported and wasn't sure if it was affecting the entire render or not.

**Comment by JosefJezek** [comment]
@stramel please send PR with solution for IE11...
